src/components/common/ExtensionsMetadata.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <button
+    :disabled="isBusy"
+    type="button"
+    @click="updateMetadata"
+    class="my-1 inline-flex justify-center w-full border border-gray-300 dark:border-darkGray-600 px-2 py-1 bg-white dark:bg-darkGray-900 text-xs font-light text-gray-700 dark:text-darkGray-100 hover:bg-gray-100 dark:hover:bg-darkGray-700 focus:outline-none focus:ring focus:ring-gray-100 dark:focus:ring-darkGray-600"
+  >
+    Update Metadata
+  </button>
+</template>
+<script lang="ts">
+import { defineComponent, computed, ref } from 'vue';
+import { useStore } from 'vuex';
+import { useMetaExtensions } from '@/hooks/useMetaExtensions';
+import { useChainInfo } from '@/hooks/useChainInfo';
+
+export default defineComponent({
+  setup() {
+    const store = useStore();
+    const apiRef = computed(() => store.getters.api);
+    const extensionsRef = computed(() => store.getters.extensions);
+
+    const { chainInfo } = useChainInfo(apiRef);
+
+    const { metaExtensions } = useMetaExtensions(apiRef, extensionsRef);
+    const selectedIndex = 0;
+    let isBusy = ref(false);
+    const updateMetadata = () => {
```

Comment:
can you make the component allow the users to install only if new metadata is available? Similar to the polkadot-js apps settings page.
https://github.com/polkadot-js/apps/blob/2d97581b173231db9b3f8dce08a5ac64b18316fa/packages/page-settings/src/Metadata/Extensions.tsx#L53

![Screen Shot 2021-07-19 at 5 09 29 PM](https://user-images.githubusercontent.com/40356749/126125775-476e6bcf-1378-44b1-b99b-eecd14d6f3c9.png)

![Screen Shot 2021-07-19 at 5 09 50 PM](https://user-images.githubusercontent.com/40356749/126125784-c1b1b568-efa4-43e0-993f-1bd9aa0ad6ca.png)

We can say `Update Metadata` and make the button clickable if the user needs to install the metadata. If not, we can say `Metadata Already Installed` and make the button disabled if the user already installed the latest chain metadata